### DISCUSSION TITLE
Rencontres SCRUM

### DISCUSSION THREAD

**Discussion by CollRen** [582-41E-MA/projet-web-2-CollRen#49] [Rencontres d'équipe]
@claraquintela @NabilDjerroud 
Voici le lien:
https://us05web.zoom.us/j/82102806567?pwd=wYjaoU2JbYyERURFCmJa1Nr4HErh1X.1

**Comment by claraquintela** [comment]
Est-ce que on utilise le meme lien aujourd'hui?

**Reply by CollRen** [reply]
Oui, on pourrait d'ailleurs toujours l'utiliser pour les srum. Le revoici: @NabilDjerroud 
https://us05web.zoom.us/j/82102806567?pwd=wYjaoU2JbYyERURFCmJa1Nr4HErh1X.1

**Reply by claraquintela** [reply]
invalid meeting ID

**Reply by CollRen** [reply]
https://us05web.zoom.us/j/87954436683?pwd=SaRDRMRMzSVfl6BvJ2CTZ9LJKx1D9t.1

**Comment by CollRen** [comment]
https://us05web.zoom.us/j/85167005800?pwd=bTaIQhjjZcgdh6VjZXgLZvPQ30RWoz.1

**Comment by CollRen** [comment]
@claraquintela 

npm run dev pour faire rouler nodemon... c'était déjà installé!

**Comment by claraquintela** [comment]
YEAH!!!!! :D

Em qui., 4 de jul. de 2024 às 11:45, René de Montigny <
***@***.***> escreveu:

**Comment by CollRen** [comment]
### SRUM du 5 juillet

1. Proposition  pour les Dossiers et fichiers
    - components/voitures/Carburant/create.js
    - components/voitures/Carburant/index.js
    - components/voitures/Carburant/update.js
    - components/voitures/Carburant/delete.js (Au besoin, ou dans carburant index)

**Reply by claraquintela** [reply]
2. Backlog: mis à jour

**Comment by claraquintela** [comment]
Lien? @CollRen

**Reply by CollRen** [reply]
https://us05web.zoom.us/j/83691491048?pwd=wVf2jQP5augttxd0JtKhcGPo3g1lFZ.1

**Comment by claraquintela** [comment]
Bonjours! 
Aurons-nous de daily aujourd'hui? (Lundi, 8/7))
@CollRen @NabilDjerroud

**Reply by CollRen** [reply]
J'aimerias bien. @NabilDjerroud  est-ce que ça t'arrangerait que les rencontres soit à 10h40 au lieu de 10h30?

**Reply by CollRen** [reply]
Lien pour la rencontre d'aujourd'hui 10h30: https://us05web.zoom.us/j/88689334151?pwd=X2ttgig4x13gyEP8BhEaAF3NCTJwCR.1

**Reply by NabilDjerroud** [reply]
moi j'ai aucun souci :D

**Comment by CollRen** [comment]
@claraquintela @NabilDjerroud 
Lien pour la rencontre d'aujourd'hui:
https://us05web.zoom.us/j/83331661558?pwd=GuhRMVxzDSKAVEZNDtJiu4VfkH00wp.1

**Reply by CollRen** [reply]
Bon, voici le lien fonctionnel: https://us05web.zoom.us/j/84784138222?pwd=qrgJ6sl0KVadom0LLaaZfP0w2P5rKD.1

**Comment by CollRen** [comment]
Bonjour à vous deux,

Est-ce que ça vous irait que la rencontre soit Remise à 11h30? Je ne pourrai pas être là à 10h30 aujourd'hui :-/

**Reply by CollRen** [reply]
@claraquintela @NabilDjerroud 

Bonjour à vous deux,

Est-ce que ça vous irait que la rencontre soit Remise à 11h30? Je ne pourrai pas être là à 10h30 aujourd'hui :-/

**Comment by claraquintela** [comment]
Pas de souci

**Reply by NabilDjerroud** [reply]
oui aucun souci @CollRen

**Comment by CollRen** [comment]
https://us05web.zoom.us/j/84724603628?pwd=21zaweF5RxDbvY9WMAGtJemeKGfX25.1

**Comment by CollRen** [comment]
Ajujourd'hui: https://us05web.zoom.us/j/86155782906?pwd=cFh17ar6rQDPN13qLx13RFBmxf8ZYT.1

**Comment by CollRen** [comment]
@NabilDjerroud  @claraquintela  Pull Request effectué pour la mise à jour de la barre de recherche

**Comment by NabilDjerroud** [comment]
@CollRen @claraquintela , y a t il une rencontre SCRUM aujourd'hui ?

**Reply by CollRen** [reply]
Oui, donnez-moi 5 minutes j'ai un problème avec mon internet...

**Reply by NabilDjerroud** [reply]
ok pas de problème René

**Comment by CollRen** [comment]
https://us05web.zoom.us/j/83895728868?pwd=W2RYcVw3YEFCJekZpFL0g1jUy4010x.1

**Comment by NabilDjerroud** [comment]
@CollRen @claraquintela Bonjour, la rencontre d'aujourd'hui est elle toujours prévue !

**Comment by CollRen** [comment]
https://us05web.zoom.us/j/83540877302?pwd=fQsgZbqFqTnjwSZz2maIBndxbM5JQL.1

**Comment by CollRen** [comment]
@NabilDjerroud @claraquintela https://us05web.zoom.us/j/85343884824?pwd=X1w8vQNpyAUJ5YTncIkHkkw8Z3afCI.1

**Reply by CollRen** [reply]
Bonjour Nabil et Clara,

Mon garçon a finalement un bon virus, il n'a pas très bien dormi et est toujours fiévreux ce matin.

On s'est organisé, ma copine et moi; je serai disponible pour une rencontre ou des questions/commentaires/etc. entre 13h et 16h.

Voudriez-vous qu'on se rencontre dans ces heures, ou bien vous faites le scrum sans moi?

**Reply by claraquintela** [reply]
Pas de probleme , René. 
On vera si on a besoin de réunion aujourd'hui. 

Prends soin de ton garçon <3

**Reply by NabilDjerroud** [reply]
Bonjour les gars, pas de soucis René, prends soin de ton garçon, prompt rétablissement à lui.

**Comment by NabilDjerroud** [comment]
@CollRen  Peut on faire une petite rencontre maintenant j'ai quelques questions s'il te plait ?

**Comment by CollRen** [comment]
@claraquintela @NabilDjerroud Bonjour à vous deux! Êtes-vous disponible pour une rencontre maintenant ?

**Reply by NabilDjerroud** [reply]
Bonjour @CollRen oui je suis disponible maintenant pour environ 40 minutes

**Reply by NabilDjerroud** [reply]
@CollRen on peut le faire ensemble pour voir si tout marche de ton coté ? car de mon coté tout fonctionne y compris les styles.

j'ai juste Utilisateur du coté admin qui ne marche pas et auquel je n'ai pas touché

**Comment by claraquintela** [comment]
Allo Rene

J’ai envoyé une message tantôt. J’ai la covid. Je me repose.
Demain peut etre si tu se passe bien.

Bonne journée

**Reply by NabilDjerroud** [reply]
Bonjour Clara, 

repose-toi bien, je te souhaite un rapide rétablissement 

Bonne journée

**Reply by CollRen** [reply]
Ok Clara prend soin de toi et on voit demain si tu vas mieux. @NabilDjerroud , on peut continuer le travail et se parler demain?

**Reply by NabilDjerroud** [reply]
ok, d'accord pas de probleme

**Comment by CollRen** [comment]
Bonjour @claraquintela et @NabilDjerroud
Premièrement, comment ça va Clara? Est-ce que tu es assez en forme pour qu'on se fasse une réunion Scrum ce matin?

**Reply by NabilDjerroud** [reply]
Bonjour @CollRen ça va bien merci et toi ?
Moi je suis prêt maintenant, par contre dans 30 minutes je dois quitter.

**Reply by CollRen** [reply]
As-tu des questions sur le code, comment ça avance de ton côté?

**Reply by NabilDjerroud** [reply]
Non de mon coté tout est beau, j'ai juste une suggestion par rapport au bouton reserver qui est dans les tuiles, je pense qu'on devrais le retirer d'ici car je pense que pour qu'une personne reserve une voiture il doit voir les détails de la voiture, ce n'est pas de la tuile qu'il va la reserver ! qu'en pensez-vous ?

**Reply by CollRen** [reply]
Je suis d'accord avec toi, ce n'est pas comme ajouter des bonbons! 

- [ ] Bouton réserver dans la page du détail

**Reply by NabilDjerroud** [reply]
sinon j'ai fait une PR en finalisant 80% du travail que j'avais à faire, est ce que ça marche pour toi ?

**Reply by NabilDjerroud** [reply]
aussi un probleme pour moi je n'ai plus de voiture à vendre elles sont toutes vendues ou reservées :D  😂 

![image](https://github.com/user-attachments/assets/c038ecad-1e45-41a9-a895-28b2311e28dc)

**Reply by CollRen** [reply]
Je vais aller voir ça en début pm, peux-tu me faire une liste des choses à vérifier/tester s'il te plait?

**Reply by NabilDjerroud** [reply]
voici une liste des choses à vérifier:

- [ ] Ajouter/supprimer une voiture au panier.
- [ ] Vider le panier
- [ ] Vérifier si le panier de chaque utilisateur n'est pas le même (lorsqu'on ajoute une voiture dans un panier, elle ne doit pas être ajoutée pour tous les utilisateurs)
- [ ] Réserver / acheter une voiture avec les formulaires respectifs (vous serez redirigés vers la page confirmation)
- [ ] les tables commandes et voitures se mettent à jour
- [ ] Une fois voiture vendue ou réservée, sa tuile est grisée avec une mention du statut de la voiture
- [ ] Ajouter Statut aux filtres afin de masquer les voitures vendues ou réserver

**Comment by claraquintela** [comment]
Bonjour, le gars

Pas encore prête à retourner. :(
Je suis desolée:(
Je me sens coupable.
Demain certainement je serais avec vous.

Encore, pardonnez-moi

**Reply by NabilDjerroud** [reply]
Bonjour @claraquintela ce n'est pas grave, repose toi bien :)

**Reply by CollRen** [reply]
Aucun soucis Clara, prend soin de toi.

**Comment by claraquintela** [comment]
Bonjour @CollRen et @NabilDjerroud 
Avons nous de daily ce matin? :)

**Reply by CollRen** [reply]
J'étais en train de vous écrire et j'ai reçu un appel pour le stage pour lequel j'ai été accepté!

Bref, je suis partant pour une rencontre ce matin.

- [ ] J'ai fait mon rebase et je suis prêt à tester tes trucs Nabil
- [ ] J'ai demandé à Maxime Pigeon de repousser la rencontre de sprint 3 afin qu'on puisse au moins se voir un fois à trois avant. Je le tiens au courant.

**Reply by CollRen** [reply]
https://us05web.zoom.us/j/86137452874?pwd=QDJz4oxoxbBsHg9ytLnaZboyJ1FPu5.1

**Reply by NabilDjerroud** [reply]
Bonjour tout le monde,

J'arrive dans 5 minutes

**Comment by claraquintela** [comment]
Excellente nouvelle ! Bravo!


Em qua., 7 de ago. de 2024 às 10:40, René de Montigny <
***@***.***> escreveu:

**Reply by CollRen** [reply]
Merci! Je suis soulagé et très content que ça fonctionne dans cette entreprise...

**Comment by CollRen** [comment]
@NabilDjerroud @claraquintela 

nouveau lien zoom: https://us05web.zoom.us/j/88455996801?pwd=JWdAl26QcsbqWtllhkcFb5MTHhIWqP.1

**Comment by CollRen** [comment]
Bonjour @claraquintela et @NabilDjerroud 
Ça vous va si on se parle plus tard, vers 15h?

Aussi, Maxime propose qu'on se voit à 11h15 vendredi au collège ça vous va aussi ?

**Reply by CollRen** [reply]
Maxime veut du même coup en profiter pour voir notre présentation

**Reply by NabilDjerroud** [reply]
Bonjour @CollRen moi je ne serais pas disponible l'après-midi, sinon pour ma part j'ai finalisé les styles et jai fait une PR,

**Reply by claraquintela** [reply]
Question; @CollRen peux tu faire la remise d'ajourd'hui?

**Reply by CollRen** [reply]
La remise pour Maxime ou Marcos ou les deux? Oui je pourrai faire ça ce soir. Ok donc on se parle ce soir tard pour finaliser d'ici à vendredi midi ?

**Reply by claraquintela** [reply]
Pour Marcos.

J'ai un test pour un stage cette après midi.....
